system-addon/lib/PrefsFeed.jsm

```diff
@@ -63,6 +66,9 @@ this.PrefsFeed = class PrefsFeed {
       values[name] = this._prefs.get(name);
     }
 
+    // Not a pref, but we need this to determine whether to show private-browsing-related stuff
+    values.isPrivateBrowsingEnabled = PrivateBrowsingUtils.enabled;
```

Comment:
@mkaply does this look right to you? Also, how would we go about watching for changes to this?

Reply:
Ok perfect, thanks!